assets/stylesheets/mobile/chat-drawer.scss

```diff
@@ -0,0 +1,63 @@
+.full-page-chat {
```

Comment:
This is not exactly BEM: there's nesting + naming isn't tied to the main class.
Could you align it more with my codepen (https://codepen.io/chapoi/pen/qBoKKGP?editors=1100)?
If there's any concerns, we can figure it out together

Reply:
@chapoi I agree 💯 this isn't BEM, but would be nice if it was. I was simply making use of the classes/markup already used. Note the template is using the existing desktop template with some conditionals for mobile (not many changes).

I wasn't sure if I should modify the markup to better accommodate BEM, as it might be a bit out of scope for this PR, as I would also need to modify some existing tests and desktop stylesheets. @jjaffeux What do you think?

Reply:
@keegangeorge Ah of course, that makes sense. I thought this was a from-scratch refactor, but if's Joffrey's fault ofc 😜 Ignore it then, carry on! ;)